core/errors.go

```diff
@@ -92,6 +92,8 @@ var errorFormatter = httperror.Formatter{
 		errInvalidAddr:                 {400, "CH161", "Address is invalid"},
 		raft.ErrAddressNotAllowed:      {400, "CH162", "Address is not allowed"},
 		raft.ErrUninitialized:          {400, "CH163", "No cluster configured"},
```

Comment:
I should've caught this before, but I think we should not use the word "configure" to describe this state, since we already use it to mean something else that's pretty specific. Can we change this to say "Cluster not initialized" or "Uninitialized cluster" or something like that?